modules/ding_wayf/ding_wayf.module

```diff
@@ -428,6 +507,14 @@ function ding_wayf_get_credentials() {
   $saml = ding_wayf_get_saml_connection();
   $saml_attributes = $saml->getAttributes();
 
+  // Check security - we need to do it here, it is the only place where we
+  // have (unspoiled) attributes directly from gatewayf.
+  $status = ding_wayf_security_check($saml_attributes);
+  /** @var bool $status */
```

Comment:
Why is this needed? `ding_wayf_security_check()` already declares that it returns a boolean.